src/screens/ChannelListScreenV2/AnonymousChannelListScreen/index.tsx

```diff
@@ -11,7 +11,7 @@ import Search from '../../ChannelListScreen/elements/Search';
 import useAnonymousChannelListScreenHook from '../../../hooks/screen/useAnonymousChannelListHook';
 import useLocalDatabaseHook from '../../../database/hooks/useLocalDatabaseHook';
 import useRootChannelListHook from '../../../hooks/screen/useRootChannelListHook';
-import {ANON_PM, ANON_POST_NOTIFICATION} from '../../../hooks/core/constant';
+import {ANONYMOUS, ANON_PM, ANON_POST_NOTIFICATION} from '../../../hooks/core/constant';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been modified to include the `ANONYMOUS` constant. This change seems fine as long as the `ANONYMOUS` constant is used somewhere in the file. If it's not used, it should be removed to avoid unnecessary imports.
```diff
- import {ANON_PM, ANON_POST_NOTIFICATION} from '../../../hooks/core/constant';
+ import {ANONYMOUS, ANON_PM, ANON_POST_NOTIFICATION} from '../../../hooks/core/constant';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->